sys/mips/conf/std.CHERI

```diff
@@ -5,10 +5,11 @@
 # $FreeBSD$
 #
 nomakeoptions	ARCH_FLAGS
-makeoptions	ARCH_FLAGS="-target mips64-unknown-freebsd -march=beri -mabi=64 -mcpu=beri -cheri -Wno-unused-command-line-argument"
+makeoptions	ARCH_FLAGS="-march=beri -mcpu=beri"
 options 	CPU_CHERI
 nooptions	COMPAT_FREEBSD32
 options 	COMPAT_FREEBSD64
+options 	BOOTVERBOSE=1
```

Comment:
Behavioural change?

Reply:
I missed that, I think I added it at some point because the purecap kernel did not produce `start_init` output, causing false timeouts in jenkins.